skirmish-report.xsl

```diff
@@ -1301,10 +1301,10 @@
     </xsl:template>
     <xsl:template name="timestamp-from-seconds">
         <xsl:param name="seconds" />
-        <xsl:variable name="abs-seconds" select="number(concat(substring(string($seconds), 1, number($seconds >= 0) * string-length(string($seconds))), substring(string($seconds * -1), 1, number(not($seconds >= 0)) * string-length(string($seconds * -1)))))" />
```

Comment:
I remembered the reason I had this here is because sometimes I'd get a battle report with a negative value for seconds.